getting-started/git-boot-camp.rst

```diff
@@ -435,8 +435,8 @@ Solution:
 
 .. _git_pr:
 
-Downloading other's patches
----------------------------
+Checking out others' pull requests
+----------------------------------
```

Comment:
```suggestion
Testing others' pull requests locally
-------------------------------------
```

Another nitpick, but if "checking out" is meant in the Git sense (rather than in the "taking a look" sense), you are technically checking out the branch, not the PR.

Reply:
True, the git operation is checking out a branch, but we also talk about "checkout out the change" or "checking out your feature," etc, even in Git terms.  The text of the section is about pull requests, and "checking out" has two meanings, both of which apply here, so I like the title as it is.